src/ort_genai_c.cpp

```diff
@@ -208,6 +208,34 @@ OgaResult* OGA_API_CALL OgaGenerator_GenerateNextToken(OgaGenerator* generator)
   OGA_CATCH
 }
 
+OgaResult* OGA_API_CALL OgaGenerator_GetOutput(const OgaGenerator* oga_generator, const char* name, OgaTensor** out) {
+  OGA_TRY
+  auto& generator = *reinterpret_cast<const Generators::Generator*>(oga_generator);
+  auto* ortvalue_output = generator.state_->GetOutput(name);
+  auto type_info = ortvalue_output->GetTensorTypeAndShapeInfo();
+  std::unique_ptr<OrtValue> ortvalue_clone = OrtValue::CreateTensor(*generator.model_->allocator_device_,
```

Comment:
the allocator_device_ can be allocator on GPU. allocator_cpu_ is the one we should use.

Reply:
Okay, I thought we need to return the logits on the same device. So, I will return the values on the CPU only.

Reply:
yes, we return the value on CPU only.